server/src/Q3Server/Properties/PublishProfiles/Q3MobileTest.pubxml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
New publishing profile to a test server. Lan and I have copied over Gareth's settings. Seems to work.